jsonnet/prometheus-operator/prometheuses-crd.json

```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
```

Comment:
Seems like rebase went wrong here.

Reply:
@paulfantom ah sorry for that. Now it should be OK